packages/api-client/src/connections/EntityApi.ts

```diff
@@ -4,21 +4,6 @@
  *
  */
 
-import type {
```

Comment:
Had to remove types due to not being able to rely on special "importing types does not mean having to register the package as a dependency" because of setting up each package properly now. Will add back in in colocate types card.

Reply:
Yeah for sure they will come back soon